src/util/ui-options.js

```diff
@@ -0,0 +1,31 @@
+// mixin.js
```

Comment:
This is in public space. Whenever you create a new file, you need to put that Copyright blah blah blah stanza up on top.

Reply:
Oh thank you! I'll add that!